sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -214,74 +210,116 @@ for cell in table.cells:
 ```
 
 ### Train a model
-Train a machine-learned model on your own form type. The resulting model will be able to extract values from the types of forms it was trained on.
-Provide the training documents as a container SAS url to your Azure Storage Blob container. See details on setting this up
+Train a machine-learned model on your own form type. The resulting model will be able to recognize values from the types of forms it was trained on.
+Provide a container SAS url to your Azure Storage Blob container where you're storing the training documents. See details on setting this up
 in the [service quickstart documentation][quickstart_training].
 
 ```python
-from azure.ai.formrecognizer import CustomFormClient
+from azure.ai.formrecognizer import FormTrainingClient
 from azure.core.credentials import AzureKeyCredential
 
-client = CustomFormClient(endpoint, AzureKeyCredential(key))
+client = FormTrainingClient(endpoint, AzureKeyCredential(key))
 
-blob_sas_url = "xxxxx"  # training documents uploaded to blob storage
+blob_sas_url = "xxx"  # training documents uploaded to blob storage
```

Comment:
container_sas_url

Reply:
sorry, should've made the change in readme too. thanks!